sys-fs/mergerfs/mergerfs-2.24.2.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs autotools
+
+DESCRIPTION="A featureful FUSE union filesystem across multiple drives"
+HOMEPAGE="https://github.com/trapexit/mergerfs"
+# tarball under download/ has prebuilt ChangeLog and AUTHORS,
+# and more importantly version.hpp,
+# so use it instead of git tag tarball
+SRC_URI="https://github.com/trapexit/mergerfs/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+xattr"
+# mergerfs bundles its own _patched_ copy of libfuse.
+# No easy way around it yet. :(
+#IUSE="${IUSE} +external-fuse"
+
+DEPEND="xattr? ( sys-apps/attr )"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-Makefile-don-t-touch-LDFLAGS.patch
+)
+
+DOCS=(
+	AUTHORS
+	ChangeLog
+	README.md
+)
+
+domake() {
+	local makeargs=( "$@" )
```

Comment:
Do you really need two vars here? Either include `"${@}"` in `makevars`, or pass it directly to `emake`.

Reply:
OK, will pass it to `emake`.